subprojects/logging/src/main/java/org/gradle/internal/logging/console/ThrottlingOutputEventListener.java

```diff
@@ -32,6 +33,8 @@
  * Queue output events to be forwarded and schedule flush when time passed or if end of build is signalled.
  */
 public class ThrottlingOutputEventListener implements OutputEventListener {
+    private final long updateNowFlushInitialDelayMs = 500L;
```

Comment:
I guess we really only ever create one instance of `ThrottlingOutputEventListener` so it's doesn't make a difference. I am fine with making it `static`.